### DISCUSSION TITLE
Treesitter is not loading the colors on large files

### DISCUSSION THREAD

**Discussion by HermesSantos** [LazyVim/LazyVim#2618] [Q&A]
if i have a large file (2k lines or even less) the treesitter just dont works.
thats in a PHP file - ![image](https://github.com/LazyVim/LazyVim/assets/94411142/d53c0c42-ce50-4dc1-982c-f61280c5ffc0)
in Vue large files it happens as well. Is there any default config wich i can desable it?

**Comment by madflow** [comment]
What is it, you would like to disable?

**Reply by HermesSantos** [reply]
i didn't see your reply, sorry. I want that the colors load properly with large file.
